src/main/java/count/TaskList.java

```diff
@@ -0,0 +1,33 @@
+package count;
+
+import java.util.ArrayList;
+
+import count.task.Task;
+
+/**
+ * The TaskList class stores the ArrayList&lt;Task&gt;
+ * allowing it to be passed by reference
+ * @author Kieran Koh Jun Wei
+ */
+public class TaskList {
+    protected ArrayList<Task> ls;
+
+    /**
+     * Constructor for TaskList if no ArrayList&lt;Task&gt; is provided
+     */
+    public TaskList() {
+        this.ls = new ArrayList<>();
+    }
+
+    /**
+     * Constructor for TaskList if an ArrayList&lt;Task&gt; is provided
+     */
+    public TaskList(ArrayList<Task> ls) {
+        this.ls = ls;
+    }
+
+    public ArrayList<Task> getList() {
```

Comment:
perhaps getTaskList would be a better name to name this? It would convey the intent of the method more